dbt/include/snowflake/macros/adapters.sql

```diff
@@ -21,7 +21,9 @@
           temporary
         {%- elif transient -%}
           transient
-        {%- endif %} table {{ relation }} {% if copy_grants and not temporary -%} copy grants {%- endif %} as
+        {%- endif %} table {{ relation }}
+        {{ get_columns_spec_ddl() }}
```

Comment:
```suggestion
        {{ get_columns_spec_ddl() }}
        {{ get_assert_columns_equivalent(sql) }}
```
This creates a validation check that columns in a sql statement match the schema config